CMSgov/mint-app
# [NOREF] Configurable email sender via environment variables

# NOREF

## Changes and Description

- Allow configuring the sender used to send emails via the `EMAIL_SENDER` environment variable

## How to test this change

- `scripts/dev up` to start the server
- Launch the app and create a model plan
- Add someone as a collaborator
- Check to see if the "added as collaborator" email is sent from the appropriate address.

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
